internal/pkg/app/handlers_v1.go

```diff
@@ -171,6 +178,14 @@ func (a *APIManager) addRouteHandler(w http.ResponseWriter, r *http.Request) {
 		resp.Respond(ctx, w, doYaml(r))
 		return
 	}
+	bearerToken := getBearerToken(r)
+	_, _, authErr := a.jwtManager.VerifyToken(ctx, bearerToken, r.URL.Path, r.Method, tid)
```

Comment:
All jwt authentication should happen here: https://github.com/xmidt-org/ears/blob/main/internal/pkg/app/middlewares.go#L70

Reply:
ok